sycl/source/detail/pi.cpp

```diff
@@ -35,6 +28,25 @@ bool piUseBackend(PiBackend Backend) {
   return Backend == Use;
 }
 
+// Definitions of the PI dispatch entries, they will be initialized
+// at their first use with piInitialize.
+#define _PI_API(api) decltype(::api) * api = nullptr;
```

Comment:
Is it not scary to have a global `api` variable?

Reply:
It will need to be globals in the long run. The PI plugins (shared libraries) are going to be discovered on the system by locating these globals, and they will connect with SYCL RT (and possibly other tools) with these globals. Remember that these globals are pointers to extern ""C" PI API.

Reply:
Yes but then why such a trivial name? It seems like a very high conflict probablility...
It could be `_sycl_backend_pi_api_` or whatever.